ansible/roles_ocp_workloads/ocp4_workload_opendatahub_old/defaults/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+silent: false
+
+# Defaults values below are for OpenShift Open Data Hub
+# via the online catalog
+
+ocp4_workload_odh_namespace: odh_project
+
+ocp4_workload_odh_namespace_display_name: Red Hat ODH
+# Channel to use for the ODH subscription
+ocp4_workload_odh_channel: legacy
+
+# Set InstallPlan approvel to Automatic? If no it is also suggested
+# to set the starting_csv to pin a specific version
+# This variable has no effect when using a catalog snapshot (always true)
+ocp4_workload_odh_automatic_install_plan_approval: true
+
+# Set a starting ClusterServiceVersion.
+# Recommended to leave empty to get latest in the channel when not using
+# a catalog snapshot.
+# Highly recommended to be set when using a catalog snapshot but can be
+# empty to get the latest available in the channel at the time when
+# the catalog snapshot got created.
+ocp4_workload_odh_starting_csv: ""
+
+# Deploy an instance of Open Data Hub
+ocp4_workload_odh_deploy: true
+
+## Settings for odh
+
+workshop_preload_repos: ""
```

Comment:
If these also are only for this role would you mind prefixing with the role name, please?